<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmComboBox</class>
 <widget class="QWidget" name="frmComboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmComboBox</string>
  </property>
  <widget class="ComboBox" name="cbox1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="ComboBox" name="cbox2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="cbox3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>61</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="cbox4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>87</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
